servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/ErrorContext.java

```diff
@@ -5,23 +5,19 @@
 
 import java.util.Locale;
 
-abstract class ErrorContext
-{
+abstract class ErrorContext {
     private final String namespaceName;
 
-    ErrorContext(final String namespaceName)
-    {
+    ErrorContext(final String namespaceName) {
         this.namespaceName = namespaceName;
     }
 
-    protected String getNamespaceName()
-    {
+    protected String getNamespaceName() {
         return this.namespaceName;
     }
 
     @Override
-    public String toString()
-    {
-        return StringUtil.isNullOrEmpty(this.namespaceName) ? null : String.format(Locale.US, "NS: %s", this.namespaceName);
+    public String toString() {
```

Comment:
[SpotBugs-P2]
com.microsoft.azure.servicebus.primitives.ErrorContext.toString() may return null

toString method return empty string will not be a code-breaking change